packages/lavapack/src/builder-runtime.js

```diff
@@ -1,10 +1,18 @@
 const { join: pathJoin } = require('path')
 const { promises: { readFile, writeFile }} = require('fs')
 const { generateKernel, makeInitStatsHook } = require('lavamoat-core')
+const { getStrictScopeTerminatorShimSrc, replaceTemplateRequire } = require('lavamoat-core/src/generateKernel')
 
 module.exports = buildRuntime
 
+async function buildRuntimeCjs() {
```

Comment:
I may be missing something here but `generateKernel` called by `buildRuntime` considers the options `debugMode`,`scuttleGlobalThis`,`scuttleGlobalThisExceptions`.
Is there some inherent difference with CommonJS modules that make them not relevant? It seems prudent to either handle it consistently or add a comment explaining the motivation behind having the functionality for one and not the other.

Reply:
The reason this was called `runtime-cjs` is not super clear to me. What I do know is that `runtime` is used for LM production builds, and MM production version leans on it and so should other programs (LM and LM packages lean on it too).

On the other hand, `runtime-cjs` seems to serve nothing within the LM project, and specifically only the development version of MM (this I learned the hard way https://github.com/MetaMask/metamask-extension/pull/17043#issue-1507832165).

The development version of MM loads without LM, but it does load a mini-lockdown version instead. It seems that `runtime-cjs` is constructed to answer that need - to allow the app to run without the massive LM protecting kernel.

This should be the answer for why `runtime` builds and injects LM kernel whereas `runtime-cjs` does not.

But I'm less sure of what was the motivation behind this decision originally, and what's that has to do with `cjs`.